src/storage/client/StorageClient.h

```diff
@@ -127,6 +127,25 @@ class StorageClient {
         std::vector<storage::cpp2::PropDef> returnCols,
         folly::EventBase* evb = nullptr);
 
+    folly::SemiFuture<StorageRpcResponse<storage::cpp2::UpdateResponse>> updateVertex(
+        GraphSpaceID space,
+        std::vector<VertexID> vertices,
```

Comment:
use &&, then there is no call rvalue construction function.

Reply:
Now it only one vertex.